changelog.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 * Update e2e test plugin to terrator-plugin.
+* New display of libraries components inside ComponentDefinitionsDrawer..
```

Comment:
`..` -> `.`

Reply:
`Rework on display of the components list.` May be?

Reply:
changed